instrumentation/active_support/lib/opentelemetry/instrumentation/active_support/span_subscriber.rb

```diff
@@ -65,16 +65,17 @@ class SpanSubscriber
 
         # rubocop:disable Metrics/ParameterLists
         def initialize(name:, tracer:, notification_payload_transform: nil, disallowed_notification_payload_keys: nil, kind: nil, span_name_formatter: nil)
-          @span_name = safe_span_name_for(span_name_formatter, name).dup.freeze
+          @name = name
```

Comment:
Please rename this attribute to `pattern` now that it will match the argument passed into AS notification subscribe. This will help us disambiguate the shadowed `name` parameter passed into the `start` and `finish` methods as well. 

https://api.rubyonrails.org/classes/ActiveSupport/Notifications.html#method-c-subscribe

Reply:
I've updated the attribute name to `@pattern` on ae89d187f0f4260bc5527800754d0928a5c14851 @arielvalentin, as requested.

I didn't touch the `name` parameter to avoid disrupting the interface. Let me know if you want this to be renamed as well.

Reply:
Yes please. This Subscriber is part of the internal API of this gem and i don't think  it is used externally at the moment.

Reply:
Thank you for confirming. It should be fixed on ed978b2dd9109abd55ad5c437d95e84f2abe472f.